101-recovery-services-backup-file-share/prereqs/.settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "PrereqResourceGroupNameSuffix": ""
```

Comment:
remove this file since the storageAccount can be in a different resourceGroup/sub

Reply:
Removed. Note that while resource group can differ, both the storage account and vault must be present in the same subscription.